frappe/utils/error.py

```diff
@@ -128,14 +128,14 @@ def reader(lnum=[lnum]):
 				value = pydoc.text.repr(getattr(evalue, name))
 
 				# render multilingual string properly
-				if type(value)==str and value.startswith(b"u'"):
+				if type(value) == str and value.startswith("u'"):
```

Comment:
Actually, would this even be a necessary validation anymore? Since python 3 unicode strings no longer start with `u`.

Reply:
yeah, there's no need for it